storm-core/src/jvm/backtype/storm/scheduler/TopologyDetails.java

```diff
@@ -57,7 +62,8 @@ public TopologyDetails(String topologyId, Map topologyConf, StormTopology topolo
         this.numWorkers = numWorkers;
     }
 
-    public TopologyDetails(String topologyId, Map topologyConf, StormTopology topology, int numWorkers, Map<ExecutorDetails, String> executorToComponents) {
+    public TopologyDetails(String topologyId, Map topologyConf, StormTopology topology,
+                           int numWorkers, Map<ExecutorDetails, String> executorToComponents) {
```

Comment:
well I made the change is the constructor signature was too long so I move it to another line for ease of reading